pyomo/core/tests/unit/test_set.py

```diff
@@ -4319,15 +4319,15 @@ def _lt_3(model, i):
 
         m = ConcreteModel()
 
-        def _validate(model, i, j):
+        def _validate_I(model, i, j):
```

Comment:
I envision that future developers looking at this code will wonder, "What is `I`?"

Reply:
In my mind, it is a simple set that we validate through the normal rule (no index).

Reply:
That is, a non-indexed set.